tests/phpunit/tests/Integrations/Jetpack.php

```diff
@@ -59,7 +59,73 @@ public function test_register_is_wpcom() {
 	 * @covers ::add_to_jetpack_sitemap
 	 */
 	public function test_add_to_jetpack_sitemap() {
-		$jetpack = new \Google\Web_Stories\Integrations\Jetpack();
+		$jetpack = $this->get_jetpack_object();
 		$this->assertEqualSets( [ Story_Post_Type::POST_TYPE_SLUG ], $jetpack->add_to_jetpack_sitemap( [] ) );
 	}
+
+	/**
+	 * @covers ::filter_api_response
+	 */
+	public function test_filter_api_response() {
+		$video_attachment_id = self::factory()->attachment->create_object(
+			[
+				'file'           => DIR_TESTDATA . '/images/test-videeo.mp4',
+				'post_parent'    => 0,
+				'post_mime_type' => \Google\Web_Stories\Integrations\Jetpack::VIDEOPRESS_MIME_TYPE,
+				'post_title'     => __METHOD__,
+			]
+		);
+		$attachment          = get_post( $video_attachment_id );
+
+		$jetpack = $this->get_jetpack_object();
+		// wp_prepare_attachment_for_js doesn't exactly match the output of media REST API, but it good enough for these tests.
+		$response = rest_ensure_response( wp_prepare_attachment_for_js( $attachment ) );
+
+		$results = $jetpack->filter_api_response( $response, $attachment );
+		$data    = $results->get_data();
+		$this->assertArrayHasKey( 'mime_type', $data );
+		$this->assertSame( $data['mime_type'], 'video/mp4' );
+	}
+
+	/**
+	 * @covers ::filter_admin_ajax_response
+	 */
+	public function test_filter_admin_ajax_response() {
+		$video_attachment_id = self::factory()->attachment->create_object(
+			[
+				'file'           => DIR_TESTDATA . '/images/test-videeo.mp4',
+				'post_parent'    => 0,
+				'post_mime_type' => \Google\Web_Stories\Integrations\Jetpack::VIDEOPRESS_MIME_TYPE,
+				'post_title'     => __METHOD__,
+			]
+		);
+		$attachment          = get_post( $video_attachment_id );
+
+		$jetpack  = $this->get_jetpack_object();
+		$response = wp_prepare_attachment_for_js( $attachment );
+
+		$data = $jetpack->filter_admin_ajax_response( $response, $attachment );
+		$this->assertArrayHasKey( 'mime', $data );
+		$this->assertSame( $data['mime'], \Google\Web_Stories\Integrations\Jetpack::VIDEOPRESS_MIME_TYPE );
+
+		$_POST = [ // phpcs:ignore WordPress.Security.NonceVerification.Missing
+			'action' => 'query-attachments',
+			'query'  => [
+				'source' => 'web_stories_editor',
+			],
+		];
+
+		$data = $jetpack->filter_admin_ajax_response( $response, $attachment );
+		$this->assertArrayHasKey( 'mime', $data );
+		$this->assertSame( $data['mime'], 'video/mp4' );
+
+		unset( $_POST ); // phpcs:ignore WordPress.Security.NonceVerification.Missing
+	}
+
+	/**
+	 * @return \Google\Web_Stories\Integrations\Jetpack
+	 */
+	protected function get_jetpack_object() {
+		return new \Google\Web_Stories\Integrations\Jetpack();
+	}
```

Comment:
What's the benefit of this?

If we really care to save some bytes, I'd probably just do `use \Google\Web_Stories\Integrations\Jetpack as Jetpack_Integration` and then everywhere `$jetpack = new Jetpack_Integration()`.

Reply:
I just personally find it easier to read. I reviewed the tests again and thought this was nice improvement. I have taken your feedback.